app/views/stations/index.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Stations Index</h1>
+
+<% @stations.each do |station| %>
+  <h3><a href="/stations/<%=station.id%>"><%= station.name %></a></h3>
+  <p><%= station.city %></p>
+  <form class="" action="/stations/<%= station.id %>/edit" method="put">
+    <input type="submit" name="_method" value="Edit">
```

Comment:
The HTML spec indicates that forms can only use `post` or `get`. When you use a `put` here, I believe it actually sends a `get` request. Which is convenient, because that's what you want: a `get` to `/stations/:id/edit` is the route to display the form to edit a station, which is what you want.

If you want this to be a button in the same way the form below is a button, it's fine to style them both as forms (though you might look into how to create a button), but make sure that this form says what it's actually going to do: change the method to `get`.